Packs/Dropbox/ModelingRules/DropboxEventCollector_1_3/DropboxEventCollector_1_3.xif

```diff
@@ -1,47 +1,28 @@
 [MODEL: dataset=dropbox_dropbox_raw]
-filter
-    json_extract_scalar(event_category, "$['.tag']") != "logins"
-| alter actor_type = json_extract_scalar(actor, "$['.tag']")
+alter
+Get_IP_Type = if(origin -> geo_location.ip_address ~= "\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}", "IPv4", origin -> geo_location.ip_address ~= "[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}", "IPv6", "Unknown"),
```

Comment:
You don't need **Get_IP_Type** lets add this logic wherever you need to-
xdm.source.ipv4 
xdm.source.ipv6

Also, in general for the query, lets add tab space when writing a query line. E,g;
| alter
       ... Some Logic....
And not-
| alter
... Some Logic....